kodi_addon_checker/xml_schema/gotham_metadata.xsd

```diff
@@ -7,31 +7,37 @@
         <xs:element name="summary" type="translatedString" minOccurs="1" maxOccurs="unbounded"/>
         <xs:element name="description" type="translatedString" minOccurs="0" maxOccurs="unbounded"/>
         <xs:element name="disclaimer" type="translatedString" minOccurs="0" maxOccurs="unbounded"/>
-        <xs:element name="platform" type="platformList" minOccurs="0"/>
-        <xs:element name="language" type="xs:string"  minOccurs="0" maxOccurs="1"/>
-        <xs:element name="license" type="xs:string"  minOccurs="0" maxOccurs="1" />
-        <xs:element name="forum" type="xs:string" minOccurs="0" maxOccurs="1"/>
-        <xs:element name="website" type="xs:string" minOccurs="0" maxOccurs="1"/>
-        <xs:element name="source" type="xs:string" minOccurs="0" maxOccurs="1"/>
-        <xs:element name="email" type="xs:string" minOccurs="0" maxOccurs="1"/>
-        <xs:element name="broken" type="xs:string" minOccurs="0" maxOccurs="1"/>
-        <xs:element name="news" type="xs:string" minOccurs="0" maxOccurs="1"/>
+        <xs:element name="platform" type="platformList" minOccurs="0" maxOccurs="1"/>
+        <xs:element name="language" type="nonEmptyString"  minOccurs="0" maxOccurs="1"/>
+        <xs:element name="license" type="nonEmptyString"  minOccurs="0" maxOccurs="1" />
+        <xs:element name="forum" type="nonEmptyString" minOccurs="0" maxOccurs="1"/>
+        <xs:element name="website" type="nonEmptyString" minOccurs="0" maxOccurs="1"/>
+        <xs:element name="source" type="nonEmptyString" minOccurs="0" maxOccurs="1"/>
+        <xs:element name="email" type="nonEmptyString" minOccurs="0" maxOccurs="1"/>
+        <xs:element name="broken" type="nonEmptyString" minOccurs="0" maxOccurs="1"/>
+        <xs:element name="news" type="nonEmptyString" minOccurs="0" maxOccurs="1"/>
         <xs:element name="reuselanguageinvoker" type="xs:boolean" minOccurs="0" maxOccurs="1"/>
         <xs:element name="assets" type="assetsList" minOccurs="0" maxOccurs="1"/>
       </xs:choice>
       <xs:attribute name="point" type="xs:string" use="required"/>
       <xs:attribute name="id" type="simpleIdentifier"/>
-      <xs:attribute name="name" type="xs:string"/>
+      <xs:attribute name="name" type="nonEmptyString"/>
     </xs:complexType>
   </xs:element>
   <xs:simpleType name="simpleIdentifier">
     <xs:restriction base="xs:string">
       <xs:pattern value="(kodi|xbmc)\.addon\.metadata"/>
     </xs:restriction>
   </xs:simpleType>
+  <xs:simpleType name="nonEmptyString">
+    <xs:restriction base="xs:string">
+      <xs:pattern value="\s*([^\s]\s*)+"/>
+      <xs:maxLength value="1500"/>  
```

Comment:
Is there a reason for 1500 or is this just what you figured would be a good size?

Reply:
Random size, tried a few numbers and looked what it'll break